__tests__/app.test.js

```diff
@@ -18,3 +18,27 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of objects, one for each topic in the database", () => {
+    return request(app)
+      .get("/api/topics/")
+      .expect(200)
+      .then(({ body: { topics } }) => {
```

Comment:
It a good idea to include a length test too to make sure you're getting the correct amount of topics in your array.